webui/react/src/utils/user.ts

```diff
@@ -1,5 +1,10 @@
 import { DetailedUser, User } from 'types';
 
-export function getDisplayName (user: DetailedUser | User | undefined): string {
+interface UserNameFields {
+  displayName?: string;
+  username: string;
```

Comment:
**question:** should `username` be optional?

Reply:
yep missed this, thanks